gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -102,6 +105,12 @@ class GateConfig extends RedisHttpSessionConfiguration {
   }
 
   @Bean
+  JedisPool jedis(@Value('${redis.connection:redis://localhost:6379}') String connection,
```

Comment:
Went with `JedisPool` as it's more commonly used across the other Spinnaker services that `spring-data-redis` (which `gate` is using).